packages/core/src/external/aws/lambda-logic/bundle-to-brief.ts

```diff
@@ -326,11 +468,11 @@ function applyResourceSpecificFilters(res: Resource): any | undefined {
     updRes.status = Array.from(
       new Set(res.clinicalStatus?.coding?.flatMap(coding => coding.code || []))
     ).join(", ");
-    if (updRes.status === "") delete updRes.status;
+    if (isUselessStatus(updRes.status)) delete updRes.status;
   }
 
   if (res.resourceType === "Immunization") {
```

Comment:
code after the cloneDeep should use `updRes`, right?

Reply:
In this case, checking the type of the initial resource allows us to keep track of types of properties inside it. If I do
`  if (updRes.resourceType === "Immunization") {`

then the types attributes after will not be known